### PR TITLE
Prep for v1.28.0

### PR DESCRIPTION
**Pull request by odow**
## TODO

 - [x] #2451
 - [x] #2470
 - [x] #2472
 - [x] #2473
 - [x] #2474
 - [x] #2475
 - [x] #2476
 - [x] #2477

## Basic

 - [x] `version` field of `Project.toml` has been updated
       - If a breaking change, increment the MAJOR field and reset others to 0
       - If adding new features, increment the MINOR field and reset PATCH to 0
       - If adding bug fixes or documentation changes, increment the PATCH field

## Documentation

 - [x] Add a new entry to `docs/src/changelog.md`, following existing style

## Tests

 - [x] https://github.com/jump-dev/MathOptInterface.jl/actions/runs/8607293206

### CODE REVIEW

**Review by blegat — CHANGES_REQUESTED** [review]
We first need to decide what to do with https://github.com/jump-dev/MathOptInterface.jl/pull/2468. See https://github.com/jump-dev/MathOptInterface.jl/issues/2450#issuecomment-2044302953

File: Project.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name = "MathOptInterface"
 uuid = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
-version = "1.27.1"
+version = "1.27.2"
```
Comment (odow) [inline Project.toml:3]:
```suggestion
version = "1.28.0"
```

File: docs/src/changelog.md [hunk 2]
```diff
@@ -7,6 +7,23 @@ CurrentModule = MathOptInterface
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.27.2 (April 11, 2024)
```
Comment (odow) [inline docs/src/changelog.md:10]:
```suggestion
## v1.28.0 (April 11, 2024)
```

File: docs/src/changelog.md [hunk 3]
```diff
@@ -7,6 +7,28 @@ CurrentModule = MathOptInterface
...
+### Added
+
+ - Added [`Bridges.Constraint.ComplexNormInfinityToSecondOrderConeBridge`](@ref) (#2451)
+
+### Fixed
+
+ - Fixed a correctness bug getting the set of a constraint that used both
+   variable and constraint bridges (#2464) (#2472)
+ - Fixed `MethodError` in some bridges when called with `Complex`-valued
+   functions (#2468)
```
Comment (odow) [inline docs/src/changelog.md:21]:
```suggestion
   functions (#2468) (#2475)
```

**Review by blegat — APPROVED** [review]

**Pull request merged**